docs/geedocs/5.3.3/answer/releaseNotes/relNotesGEE5_3_3.html

```diff
@@ -109,9 +109,9 @@ <h1>Release notes: Open GEE 5.3.3<a class="headerlink" href="#release-notes-open
 </tr>
 </thead>
 <tbody valign="top">
-<tr class="row-even"><td>Issue Number</td>
-<td>Issue Description</td>
-<td>Issue Resolution</td>
+<tr class="row-even"><td>1202</td>
```

Comment:
Since we are now using Sphinx based documentation, we need to update the .rst, build the files and make them part of this PR. See this [wiki document](https://github.com/google/earthenterprise/wiki/Adding-and-Editing-Content-to-the-GEE-Documentationl) for how to. This [other wiki document](https://github.com/google/earthenterprise/wiki/Development:-Release-Process) reference notes how to update release notes under section `7. Update documentation`

```7. Update documentation

Make the following documentation updates and then commit them to the release branch.

Create release notes for the new release. Pattern it after the existing release notes earthenterprise/docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_2.rst.
Include the new rst file in earthenterprise/docs/geedocs/docsrc/answer/releaseNotes.rst.
Update the copyright with the current year in earthenterprise/docs/geedocs/docsrc/conf.py. Example: if the year of the update is 2018 and copyright year notice was '2016-2017' change it to '2016-2018'.
Generate html for documentation (Whenever there is a change in rst, we need to generate html and commit the changes)
$ cd earthenterprise/docs/geedocs/docsrc
$ rm -rf build
$ make html
$
you will see that based on the current release number, the folder will be created under earthenterprise/docs/geedocs```